src/fileio.jl

```diff
@@ -34,13 +36,22 @@ function load(s::Stream{format"LAS"})
     header, pointdata
 end
 
+function load(f::File{format"LAZ"})
+    # read las from laszip, which decompresses to stdout
+    open(`laszip -olas -stdout -i "$(filename(f))"`) do s
```

Comment:
Are the double quotes necessary here?  Julia backticks should handle token splitting in an intelligent way so that this is safe from whitespace separated names.